app/src/main/java/com/example/campusguide/directions/EmptyDirectionsGuard.kt

```diff
@@ -1,20 +1,18 @@
 package com.example.campusguide.directions
 
-import android.app.Activity
-import android.app.AlertDialog
-import androidx.appcompat.app.AppCompatActivity
-import com.example.campusguide.utils.MessageDialogFragment
+import com.example.campusguide.utils.ErrorListener
 
-class EmptyDirectionsGuard constructor(private val activity: AppCompatActivity, private val wrapped: GetDirectionsDialogFragment.DirectionsDialogConfirmationListener)
-    : GetDirectionsDialogFragment.DirectionsDialogConfirmationListener {
+class EmptyDirectionsGuard constructor(private val wrapped: DirectionsDialogConfirmationListener, private val errorListener: ErrorListener)
```

Comment:
Is this input validation for the directions?

Reply:
Yep, this opens a simple message box if the user leaves start or end blank